src/components/Header.test.js

```diff
@@ -0,0 +1,14 @@
+// import {screen, render} from '@testing-library/react';
+// import Header from './Header';
+// import userEvent from '@testing-library/user-event';
+// import { BrowserRouter } from 'react-router-dom';
+
+// describe("<Header/>", () => {
+//     test("header renders without error", () => {
+//         render(<Header/>)
+//         const indexLink = screen.getByText(/Header for cat Tinder/i)
+//         expect(indexLink).toBeInTheDocument()
+
+
+//     })
+// })
```

Comment:
Remove this file and bring it back for the next branch. best to not have it on the main branch even as a comment. Feel free to store it somewhere else.